### DISCUSSION TITLE
Distributed computing for lifelines?

### DISCUSSION THREAD

**Discussion by MarkPundurs** [CamDavidsonPilon/lifelines#1211] [Q&A] [answered]
It appears that lifelines is built to run on a single machine. Is there a way to distribute the computations across multiple servers in a cluster?

**Comment by CamDavidsonPilon** [comment, marked as answer]
Hi @MarkPundurs - not natively though. I don't know much about how modern distributed system handle sequential algorithms (lifelines internals use second-order minimization routines). Training _multiple_ models across a cluster (one per machine) is possible, though.

**Reply by MarkPundurs** [reply]
Thanks, @CamDavidsonPilon !
